pubspec.yaml

```diff
@@ -20,6 +19,7 @@ dependencies:
   analyzer: '>=0.35.0 <0.37.0'
   build: '>0.12.7 <2.0.0'
   built_value: ^6.1.0
+  build_config: '>=0.3.1 <5.0.0'
```

Comment:
Oh, if there was no constraint previously then we don't need to do this change. Probably a new version needs to be published?

Reply:
How do I publish it for them, Jake? I need to get this working today!

Reply:
When I run TOH 6 my `webdev` succeeds, I get no messages, and my `http://localhost:53322/index.html` keeps `loading...`. I feel like I am really close!

Reply:
You can't publish a package for somebody - they have to add you to owners.

You can try depending on this package by git repo, I think that looks like this:

```
dependencies:
  pageloader:
    git: https://github.com/google/pageloader.git
```

Reply:
Just tried that Jake, the behavior didn't change, the webpage is still loading. Any other ideas?

Reply:
Jake, the real issue is that I don't know what the problem is. How do I identify it? Which tests should I run?

Reply:
@DrMarcII we need your help!

Reply:
> if there was no constraint previously then we don't need to do this change. Probably a new version needs to be published?

There really should be a constraint. Removing it was a mistake.

https://github.com/dart-lang/build/tree/9a8ac4bfa883ebc5cf2bc3f76b6bf1393bcb9a46/build_config#publishing-buildyaml-files